### PR TITLE
Auto case: convert guest with vmware non-admin user

### PR DESCRIPTION
**Pull request by mxie91**
Signed-off-by: Ming Xie <mxie@redhat.com>

### CODE REVIEW

**Comment by mxie91** [comment]
# avocado run --vt-type v2v function_test_esx.positive_test.non_admin_user.esx_80.libvirt.it_vddk.vpx_uri
No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : 99c39ec587cc8c533ea49db643952c5816f6e883
JOB LOG    : /root/code/avocado/avocado/{{ base_dir }}/job-results/job-2022-11-30T13.57-99c39ec/job.log
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.non_admin_user.esx_80.libvirt.it_vddk.vpx_uri: STARTED
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.non_admin_user.esx_80.libvirt.it_vddk.vpx_uri: PASS (306.70 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/code/avocado/avocado/{{ base_dir }}/job-results/job-2022-11-30T13.57-99c39ec/results.html
JOB TIME   : 308.23 s


# cat  /root/code/avocado/avocado/\{\{\ base_dir\ \}\}/job-results/job-2022-11-30T13.57-99c39ec/test-results/1-type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.non_admin_user.esx_80.libvirt.it_vddk.vpx_uri/debug.log  |grep 'virt-v2v -i libvirt'
[stdlog] 2022-11-30 13:57:47,108 avocado.utils.process INFO | Running '/bin/virt-v2v -i libvirt -ic vpx://non-admin%40vsphere.local@10.73.212.149/data/10.73.212.36/?no_verify=1 -o libvirt -of raw -os v2v_dir --mac 00:50:56:af:93:f0:bridge:virbr0 esx8.0-rhel8.7-x86_64 -it vddk -io vddk-libdir=/root/vddk_libdir/latest -io vddk-thumbprint=D1:03:96:7E:11:3D:7C:4C:B6:50:28:1B:63:74:B5:40:5F:9D:9F:94 -on esx8.0-rhel8.7-x86_640WTW -v -x -ip /tmp/v2v_vpx_passwd'
[stdlog] 2022-11-30 14:02:10,249 avocado.utils.process INFO | Command '/bin/virt-v2v -i libvirt -ic vpx://non-admin%40vsphere.local@10.73.212.149/data/10.73.212.36/?no_verify=1 -o libvirt -of raw -os v2v_dir --mac 00:50:56:af:93:f0:bridge:virbr0 esx8.0-rhel8.7-x86_64 -it vddk -io vddk-libdir=/root/vddk_libdir/latest -io vddk-thumbprint=D1:03:96:7E:11:3D:7C:4C:B6:50:28:1B:63:74:B5:40:5F:9D:9F:94 -on esx8.0-rhel8.7-x86_640WTW -v -x -ip /tmp/v2v_vpx_passwd' finished with 0 after 263.138120945s

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -779,6 +781,11 @@
             main_vm = VM_NAME_ESX70_RHEL8_V2V_EXAMPLE
             vddk_thumbprint = 'AA:2B:AD'
             msg_content_yes = "nbdkit: vddk[1]: error: Please verify whether the "thumbprint" parameter"
+        - non_admin_user:
+            only esx_80
+            main_vm = VM_NAME_NON_ADMIN_V2V_EXAMPLE
+            checkpoint = 'non_admin_user'
+            vpx_user = VMWARE_NON_ADMIN_USER_EXAMPLE
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:788]: This line changes to 'vpx_username = VMWARE_NON_ADMIN_USER_EXAMPLE' should work.
The change in .py file is not necessary.

Comment (mxie91) [reply v2v/tests/cfg/function_test_esx.cfg]: Updated, pls check again, thanks!

File: v2v/tests/src/function_test_esx.py [hunk 2]
```diff
@@ -48,6 +48,7 @@ def run(test, params, env):
     vpx_hostname = params.get('vpx_hostname')
     vpx_passwd = params.get("vpx_password")
     esxi_host = esx_ip = params.get('esx_hostname')
+    vpx_username = params.get('vpx_username')
```
Comment (xiaodwan) [inline v2v/tests/src/function_test_esx.py:51]: This line is not necessary.

File: v2v/tests/src/function_test_esx.py [hunk 3]
```diff
@@ -659,6 +660,7 @@ def vm_check(status_error):
             'rhv_upload_opts': rhv_upload_opts,
             'oo_json_disk_pattern': json_disk_pattern,
             'cmd_has_ip': cmd_has_ip,
+            'vpx_username': vpx_username,
```
Comment (xiaodwan) [inline v2v/tests/src/function_test_esx.py:663]: This line is not necessary.

File: v2v/tests/src/function_test_esx.py (hunk 2 above)
Comment (mxie91) [reply v2v/tests/src/function_test_esx.py]: updated, pls check again, thanks!

File: v2v/tests/src/function_test_esx.py (hunk 3 above)
Comment (mxie91) [reply v2v/tests/src/function_test_esx.py]: done

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**